addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -160,6 +160,10 @@ const Wysiwyg = Widget.extend({
         this.$editable.data('oe-model', options.recordInfo.res_model);
         this.$editable.data('oe-id', options.recordInfo.res_id);
         document.addEventListener('mousedown', this._onDocumentMousedown, true);
+        const saveButton = this.options.document.querySelector('.o_form_button_save');
+        saveButton && saveButton.addEventListener('click', () => {
+            document.getSelection().removeAllRanges();
+        });
```

Comment:
Hello @visp-odoo ,this is not the right place to make the change. If this code targets closing the toolbar then i believe `updateValue` on [line](https://github.com/odoo/odoo/blob/16.0/addons/web_editor/static/src/js/backend/html_field.js#L304) is the right place to do it. As once the value is updated to editor and isDirty is set to false indicating the value is updated. then we can check whether it still holds a selection or not.By doing this it will save a lot of efforts of adding event or sort.
*Also whenever you add an event listener best practice is to also remove eventListener.

Reply:
Done changes :+1: